docs/topics/installing-vs-code-plugin.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-plugin-guide/master.adoc
+
+
+[id="installing-vs-code-plugin_{context}"]
+= Installing the {VscodePluginName} extension
+
+You can install the {VscodePluginName} ({VscodePluginShortName}) extension on Linux, Windows, or macOS.
+
+As of June 2021, VS Code has been tested on the following platforms:
+
+* Linux (Red Hat): Red Hat Enterprise Linux 7, CentOS 8, Fedora 24
+* Linux (Debian): Ubuntu Desktop 16.04, Debian 9
+* Windows 7 (with .NET Framework 4.5.2), 8.0, 8.1 and 10 (32-bit and 64-bit)
+* OS X El Capitan (10.11+)
+
+.Prerequisites
+
```

Comment:
Your prerequisites should be for the VS Code extension, not VS Code. Remember that we do not do documentation for third parties.